llama_stack/providers/datatypes.py

```diff
@@ -202,10 +202,15 @@ def provider_data_validator(self) -> Optional[str]:
         return self.adapter.provider_data_validator
 
 
-def remote_provider_spec(api: Api, adapter: AdapterSpec) -> RemoteProviderSpec:
+def remote_provider_spec(
+    api: Api, adapter: AdapterSpec, api_dependencies: Optional[List[Api]] = None
+) -> RemoteProviderSpec:
+    if api_dependencies is None:
+        api_dependencies = []
     return RemoteProviderSpec(
         api=api,
         provider_type=f"remote::{adapter.adapter_type}",
         config_class=adapter.config_class,
         adapter=adapter,
+        api_dependencies=api_dependencies,
```

Comment:
nit: just `api_dependencies or []` is shorter

Reply:
done.